.github/workflows/wheels.yml

```diff
@@ -71,17 +83,33 @@ jobs:
           fetch-tags: true
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.20.0
+        uses: pypa/cibuildwheel@v2.22.0
         env:
           CIBW_BUILD: ${{ matrix.build }}
-          CIBW_BEFORE_ALL_LINUX: yum install -y bzip2-devel flac-devel netcdf-devel
-          CIBW_BEFORE_ALL_MACOS: brew install bzip2 flac netcdf
-          CIBW_BEFORE_BUILD: ./deps/install_boost.sh
+          CIBW_BEFORE_ALL_LINUX: yum install -y bzip2-devel netcdf-devel
+          CIBW_BEFORE_ALL_MACOS: echo "" > /dev/null
```

Comment:
I would try to build with netcdf if possible.  There are a good number of Mac users that would use these wheels for streaming data from readout boards to KST or similar.

Reply:
Sounds good, [wheel build attempt here](https://github.com/tskisner/spt3g_software/actions/runs/13420593679).  I didn't know what netcdf was used for.  Note that netcdf depends on hdf5, which for homebrew packages is built with gcc-14.  If this causes problems, we could try to build our whole stack with homebrew gcc-14 rather than building all with clang as we are currently doing.  Let's see what happens.

Reply:
Ok, linking to homebrew netcdf seemed to work in the build (not sure if any of the unit tests exercise those features).  Note that I excluded the homebrew bzip2 package since that is "keg only" and not placed into `/usr/local/lib` (to avoid conflicts with MacOS provided bzip2).  So we have always been linking to the system bzip2 and would need to pass extra flags to cmake if we wanted to use the homebrew one.

Reply:
I think the only tests in the dfmux library are to make sure that compiled components import correctly.  Could add a test to make sure netcdfdump actually works, but I think if the import test is passing and netcdf compiles and links, then we're ok.